[Tags] Error importing light theme in dist

<!--- Provide a general summary of the issue in the Title above -->
There is an error in the relative import for tag/constants.js resulting in this error when importing StatefulSelect
`./node_modules/baseui/tag/constants.js
Module not found: Error: Can't resolve '../../src/themes/light-theme' in '/Users/todd.levin/Uber/ihub/node_modules/baseui/tag`
<!--
    Thank you very much for contributing to Base UI by creating an issue! ❤️
    To avoid duplicate issues we ask you to check off the following list.
-->

* [x] I have searched the [issues](https://github.com/uber-web/baseui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
Importing Stateful select returns no error
<!---
    Describe what should happen.
-->

## Current Behavior
Error as linked above
<!---
    Describe what happens instead of the expected behavior.
-->

## Context

<!---
    What are you trying to accomplish? How has this issue affected you?
    Providing context helps us come up with a solution that is most useful in the real world.
-->

## Your Environment

<!--- Include as many relevant details about the environment with which you experienced the bug. -->

| Tech    | Version |
| ------- | ------- |
| Base UI | v1.9.0  |
| React   |   16      |
| browser |    Chrome     |
| node.js |    8.11     |
| etc.    |         |

--- a/src/tag/constants.js
+++ b/src/tag/constants.js
@@ -6,7 +6,7 @@ LICENSE file in the root directory of this source tree.
 */
 // @flow
 /* eslint-disable import/prefer-default-export */
-import DEFAULT_THEME from '../../src/themes/light-theme';
+import DEFAULT_THEME from '../themes/light-theme';
 
 export const STYLE = {
   primary: DEFAULT_THEME.colors.primary470,

